src/js/tabs/osd.js

```diff
@@ -142,37 +142,38 @@ FONT.parseMCMFontFile = function(dataFontFile) {
         console.debug(msg);
         Promise.reject(msg);
     }
-    const character_bits = [];
-    const character_bytes = [];
+    const characterBits = [];
+    const characterBytes = [];
     // hexstring is for debugging
     FONT.data.hexstring = [];
-    const pushChar = function() {
+    const pushChar = function(fontCharacterBytes, fontCharacterBits) {
```

Comment:
Now that you have removed the need for `pushChar()` to be inside `parseMCMFontFile()`'s closure you can just as well move it outside of the function definition to make the code a bit easier to read.

Reply:
Done!